broker/src/main/java/io/camunda/zeebe/broker/jobstream/YieldingJobStreamErrorHandler.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.jobstream;
+
+import io.camunda.zeebe.protocol.impl.stream.job.ActivatedJob;
+import io.camunda.zeebe.protocol.record.intent.JobIntent;
+import io.camunda.zeebe.stream.api.scheduling.TaskResultBuilder;
+
+public class YieldingJobStreamErrorHandler implements JobStreamErrorHandler {
+  @Override
+  public void handleError(
+      final ActivatedJob job, final Throwable error, final TaskResultBuilder resultBuilder) {
```

Comment:
:thinking:  I think it would be good to log the `error` so that users are aware of why a job was yielded. It would also be good for debugging.

You can use the `JOB_STREAM` logger available [here](https://github.com/camunda/zeebe/blob/035d13bb83cf9ae325d7a895e4191b4686515b3f/broker/src/main/java/io/camunda/zeebe/broker/Loggers.java#L30).

Reply:
Makes total sense, updated! Please check how I logged it and let me know if it looks good.